src/applications/static-pages/cta-widget/ctaWidgets.js

```diff
@@ -312,12 +312,12 @@ export const ctaWidgetsLookup = {
   },
   [CTA_WIDGET_TYPES.VIEW_APPOINTMENTS]: {
     id: CTA_WIDGET_TYPES.VIEW_APPOINTMENTS,
-    deriveToolUrlDetails: authenticatedWithSSOe => ({
-      url: mhvUrl(authenticatedWithSSOe, 'appointments'),
+    deriveToolUrlDetails: () => ({
+      url: '/my-health/appointments',
       redirect: false,
     }),
-    hasRequiredMhvAccount: accountLevel => accountLevel === 'Premium',
-    isHealthTool: true,
```

Comment:
There's a `getHealthToolContent` function that we don't want to run since it just shows MHV account or portal messages, neither of which we want. Configured to not use that stuff.

Longer term the logic around these config elements should be removed, but that's out of scope for this ticket